lib/osv/ecosystems.py

```diff
@@ -310,23 +310,74 @@ def enumerate_versions(self, package, introduced, fixed, limits=None):
     return self._get_affected_versions(versions, introduced, fixed, limits)
 
 
+class Debian(Ecosystem):
+  """Debian ecosystem"""
+
+  _API_PACKAGE_URL = 'https://snapshot.debian.org/mr/package/{package}/'
+  versions_to_idx = None
+  debian_release_ver: str
+
+  def __init__(self, debian_release_ver: str):
+    self.debian_release_ver = debian_release_ver
+    debian_version_cache.initiate_from_cloud_cache()
+
+  def sort_key(self, version):
+    if self.versions_to_idx is None:
```

Comment:
We actually need to implement version sorting independently from the `enumerate_versions` method after all. This can come in a separate PR. 

Our OSV evaluation algorithm relies on this for expanding the individual versions from the events list: https://github.com/google/osv/blob/367d977409547d62713d15b59dec50f0bf1228d5/lib/osv/impact.py#L390

Reply:
Resolved using univer's debian.py library